repair/repair_task.hh

```diff
@@ -56,7 +56,34 @@ protected:
 };
 
 class repair_module : public task_manager::module {
+    // TODO Deexie:
```

Comment:
Excuse me this part, but many things happen here and I wanted to keep everything that needs to be thought through in one place as well as give you an easy insight to all that concerns.